packages/react-storage/src/components/StorageBrowser/context/controls/Navigate/Navigate.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+
+const INITIAL_STATE: NavigateState = {
+  getData: () => undefined,
+  initial: 'Home',
+  location: {
+    current: undefined,
+    isLoadingInitialData: false,
+    previousLocations: undefined,
+  },
+  locations: {
+    isLoadingInitialData: false,
+  },
```

Comment:
Just wondering, is `locations` only storing a status for now? Is it going to store other things later?